backend/drive/drive.go

```diff
@@ -199,7 +199,7 @@ func init() {
 				m.Set("root_folder_id", "appDataFolder")
 			}
 
-			if opt.ServiceAccountFile == "" {
+			if opt.ServiceAccountFile == "" || len(opt.ServiceAccountCredentials) == 0 {
```

Comment:
I think that should probably say

    if opt.ServiceAccountFile == "" && opt.ServiceAccountCredentials == "" {

so if "service account file is empty" and "service account credentials are empty" then do the oauth

Does that seem right to you?

Reply:
Yeah, You're Correct.

Mistake from My Side. Fixed It Now